android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -312,6 +313,37 @@ public LngLat coordinatesAtScreenPosition(double screenX, double screenY) {
     }
 
     /**
+     * Construct a collection of drawable map features.
+     * @param name The name of the data collection. Once added to a map, features from this
+     * {@code MapData} will be available from a data source with this name, just like a data source
+     * specified in a scene file. You cannot create more than one data source with the same name.
+     * If you call {@code addDataLayer} with the same name more than once, the same {@code MapData}
+     * object will be returned.
+     */
+    public MapData addDataLayer(String name) {
+        MapData mapData = clientDataSources.get(name);
+        if (mapData != null) {
+            return mapData;
+        }
+        long pointer = nativeAddDataSource(name);
+        if (pointer <= 0) {
+            throw new RuntimeException("Unable to create new data source");
+        }
+        mapData = new MapData(name, pointer, this);
+        clientDataSources.put(name, mapData);
+        return mapData;
+    }
+
+    /**
+     * For package-internal use only; remove a {@code MapData} from this map
+     * @param mapData The {@code MapData} to remove
+     */
+    void removeDataLayer(MapData mapData) {
```

Comment:
I agree either one seems fine (but not both) so perhaps using the convention Android developers have been trained to expect and are using in existing mapping apps should be the tie-breaker as long as it does not go against the architecture of Tangram ES.